airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/schemas.py

```diff
@@ -0,0 +1,178 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+from typing import Any, List, MutableMapping, Type
+
+from pydantic import BaseModel
+
+
+class BaseSchemaModel(BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
+        @classmethod
+        def schema_extra(cls, schema: MutableMapping[str, Any], model: Type["BaseModel"]) -> None:
+            # Modify pydantic generated jsonschema.
+            # Remove "title" and "description" fields to reduce size.
+            schema.pop("title", None)
+            schema.pop("description", None)
+            # Remove required section so any missing attribute from API wont break object validation.
+            schema.pop("required", None)
+            for name, prop in schema.get("properties", {}).items():
+                prop.pop("title", None)
+                prop.pop("description", None)
+                allow_none = model.__fields__[name].allow_none
+                # Pydantic doesnt treat Union[None, Any] type correctly when
+                # generation jsonschema so we cant set field as nullable (i.e.
+                # field that can have either null and non-null values),
+                # generate this jsonschema value manually.
+                if "type" in prop:
+                    if allow_none:
+                        prop["type"] = ["null", prop["type"]]
+
+
+class Application(BaseSchemaModel):
+    id: str
+    type: str
+    candidateId: str = None
```

Comment:
You can use AllOptional metaclass from source_braintree/schemas/common.py, it would make all fields optional.